src/Turing.jl

```diff
@@ -87,7 +87,8 @@ using Turing.Traces
 
 # Turing essentials - modelling macros and inference algorithms
 export @model, @~, @VarName                   # modelling
-export HMC, SGLD, SGHMC, HMCDA, NUTS, IS, SMC, CSMC, PG, PMMH, IPMCMC, Gibbs # sampling
+export MH, HMC, SGLD, SGHMC, HMCDA, NUTS      # sampling
+export IS, SMC, CSMC, PG, PIMH, PMMH, IPMCMC, Gibbs # sampling
```

Comment:
Can we make the comments more specific in stead of just say `sampling`? And probably make them aligned.